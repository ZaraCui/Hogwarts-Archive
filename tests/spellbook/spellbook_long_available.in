ADD COLLECTION tests/data/books_small.csv
SPELLBOOK 1002 LONG
EXIT

