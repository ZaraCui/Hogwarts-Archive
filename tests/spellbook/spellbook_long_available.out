user: 12 spellbooks successfully added.

user: 1002: Dark Arts Basics (Salazar Slytherin, Curse)
Currently available.

user: Ending Archive process.
